libs/dev-tools/src/main/kotlin/uk/gov/justice/digital/hmpps/ReflectionUtils.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps
+
+fun Any.set(field: String, value: Any) {
```

Comment:
Fyi there is a Spring utils method for this: 
`org.springframework.data.util.ReflectionUtils.setField(f, this, value)`

This use case simple enough so happy to leave it for now, but if we start using reflection more in future it might be worth leaning on the Spring classes, as they appear to handle some edge cases and throw better exceptions.